src/lavinmq/http/controller/shovels.cr

```diff
@@ -0,0 +1,67 @@
+require "../controller.cr"
+
+module LavinMQ
+  module HTTP
+    class ShovelsController < Controller
+      include StatsHelpers
+
+      private def register_routes
+        get "/api/shovels" do |context, _params|
+          itrs = vhosts(user(context)).flat_map do |v|
+            v.shovels.each_value
+          end
+          page(context, itrs)
+        end
+
+        get "/api/shovels/:vhost" do |context, params|
+          with_vhost(context, params) do |vhost|
+            page(context, @amqp_server.vhosts[vhost].shovels.each_value)
```

Comment:
We can and should use the `vhost` instance from `with_vhost` (so wrong already in main controller)
```suggestion
            page(context, vhost.shovels.each_value)
```